### DISCUSSION TITLE
How to boot .img in UTM on Mac

### DISCUSSION THREAD

**Discussion by TheOrdinaryWizard** [utmapp/UTM#5309] [UTM Help (macOS)]
Whenever i try to boot a .img image it boots to the UEFI shell and doesn't boot the installer. I've seen a similar discussion to this which suggested creating a non-removable drive that uses the image but that hasn't worked. Help would be appreciated thanks. :)

**Comment by macuserguru** [comment]
I have similar problem with concrete Linux distributions -->
https://github.com/utmapp/UTM/discussions/5304

All tips in some Linux forums I found refer to an installation on a computer hardware itself, not for VMs

**Reply by TheOrdinaryWizard** [reply]
Same, but I think for hardware you can jsut write to the sd card, but for VMs I'm not sure if you can mount a sd card like that since it doesn't show up on my desktop or anything

**Reply by macuserguru** [reply]
I have not yet found an understandable 'recipe' - especially one that works so easily on the M1 Mac.
